addon/components/frost-fixed-table.js

```diff
@@ -278,6 +329,32 @@ export default Component.extend({
     this.$(`${bodyMiddleSelector} .frost-table-row`).css({width: `${width}px`})
   },
 
+  setupLeftAndRightWidths () {
+    const sides = [
+      {
+        header: this.get('headerLeftSelector'),
+        body: this.get('bodyLeftSelector')
+      }, {
+        header: this.get('headerRightSelector'),
+        body: this.get('bodyRightSelector')
+      }
+    ]
+    for (let side of sides) {
+      const headerCells = this.$(`${side.header} .frost-table-header-cell`)
+      for (let pos = 0; pos < headerCells.length; ++pos) {
```

Comment:
huh? As far as I know there is no difference https://stackoverflow.com/questions/29885719/i-vs-i-in-a-javascript-for-loop. Just a habit of mine because post-increment creates and return a copy of the original value so as tiny as it may be, uses less memory. The pos + 1 is due to the fact that first element is selected with nth-child(1) where as the array starts at index 0